src/frontend/src/components/comment/Comment.vue

```diff
@@ -4,44 +4,47 @@
     </v-flex> 
     위의 세 줄은, 나중에 대댓글 쓸때 인덴트 넣어써 쓰면 됨
     -->
-  <v-card style="padding:8px" flat max-width="400">
-    <v-card-title class="pa-1">
-      <div class="mr-3">
+  <v-card flat max-width="400">
+    <v-card-title class="pa-1 pb-0">
+      <div class="ml-2" style="font-size:14px; color:black;">
         {{ comment.nickname }}
       </div>
       <v-spacer />
-      <div>
-        <created-date :createdDate="comment.createdDate" />
-      </div>
+      <v-card-actions class="pa-0">
```

Comment:
```suggestion
      <v-card-actions class="mr-1 pa-0">
```

![image](https://user-images.githubusercontent.com/42054054/91323589-0501cb80-e7fc-11ea-8c33-26ff8168fdff.png)


사진에 표시된 공간이 조금 여유있으면 좋을것같아요!

Reply:
홀리몰리!! 점점점 있는걸 중점적으로 보다가 놓쳤네요!
알려주셔서 고맙습니다 🙂 
다녀올게요~

Reply:
저는 등록시간 글씨크기가 쪼끔만 더 작아지면 좋을 것 같아요!

Reply:
으음 저도 그렇게 생각하긴 했는데, 우선은 게시글이랑 사이즈를 맞추어 두었어요!

Reply:
방금 10px로 확인해보니까 본문도 같이 더 줄어들면 이쁠 거 같은데 그렇게 변경해볼게요!